app/res/layout/blue_outlined_button.xml

```diff
@@ -6,5 +6,6 @@
     android:layout_gravity="right"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/5780bb551b517d632277741d48e3617b81df0e13/app/res/layout/blue_outlined_button.xml#L6"></span>Use "<code>end</code>" instead of "<code>right</code>" to ensure correct behavior in right-to-left locales</td>
    </tr>
  </tbody>
</table>